src/javascript/app/base/header.js

```diff
@@ -622,17 +685,78 @@ const Header = (() => {
+        const real_accounts               = all_login_ids.filter((loginid) => !/^VRT/.test(loginid));
+        const has_real_account            = real_accounts.length > 0;
+        const has_mf_account              = all_login_ids.some(loginid => loginid.startsWith('MF'));
+        const has_non_eu_account          = all_login_ids.some(loginid => loginid.startsWith('CR'));
+        const current_active_login        = Client.get('loginid');
+        const is_virtual                  = current_active_login.startsWith('VRTC');
+        const manage_acc_btn              = document.getElementById('account__switcher-manage');
+        const new_account_adder_deriv     = document.getElementById('account__switcher-new-account-deriv');
+        const new_account_adder_eu        = document.getElementById('account__switcher-new-account-eu');
+        const traders_hub_link            = document.getElementById('account__switcher-cfd');
+        const account_switcher_seperator  = document.getElementById('cfd-link-seperator');
+        const add_account_text_normal     = document.getElementById('add-account-text-normal');
+        const add_account_text_eu_country = document.getElementById('add-account-text-eu');
+        const multiplier_text             = localize('Multipliers');
+        const showTradersHubLink = (show) => {
+            traders_hub_link.style.display            = show ? 'flex' : 'none';
+            account_switcher_seperator.style.display  = show ? 'block' : 'none';
+        };
 
+        if (current_active_login.startsWith('MF')) {
+            $(`<span class="header__acc-display-text">${multiplier_text}</span>`).insertAfter('#header__acc-balance');
+        }
+        if (isEuCountry()) {
+            add_account_text_normal.style.display     = 'none';
+        } else {
+            add_account_text_eu_country.style.display = 'none';
+        }
+        if (has_real_account) showTradersHubLink(true);
+        if (is_virtual || !has_real_account)  {
+            manage_acc_btn.setVisibility(0);
+        }
+        if (has_real_account && !is_virtual) {
+            manage_acc_btn.setVisibility(1);
+        }
+        // Account adder logic
+        if (!has_real_account) {
+            new_account_adder_deriv.style.display     = 'flex';
+            new_account_adder_eu.style.display        = 'flex';
+        } else {
+            if (has_mf_account && has_non_eu_account) {
+                new_account_adder_deriv.style.display = 'none';
+                new_account_adder_eu.style.display    = 'none';
+            }
+            if (has_mf_account && !has_non_eu_account) {
+                new_account_adder_eu.style.display    = 'none';
+                new_account_adder_deriv.style.display = 'flex';
+            }
+            if (has_non_eu_account && !has_mf_account) {
+                new_account_adder_deriv.style.display = 'none';
+                new_account_adder_eu.style.display    = 'flex';
+            }
+        }
```

Comment:
```suggestion
        new_account_adder_deriv.style.display     = 'flex';
        new_account_adder_eu.style.display        = 'flex';
        if (has_real_account) {
                if (has_mf_account && has_non_eu_account) {
                    new_account_adder_deriv.style.display = 'none';
                    new_account_adder_eu.style.display    = 'none';
                } else if (has_mf_account && !has_non_eu_account) {
                    new_account_adder_eu.style.display    = 'none';
                 } else if (!has_mf_account && has_non_eu_account) {
                    new_account_adder_deriv.style.display = 'none';
                }
        }
```
@henry-deriv could you please check if this shorter version is gonna work? )